src/java.desktop/share/classes/javax/swing/text/html/FormView.java

```diff
@@ -193,7 +193,7 @@ public FormView(Element elem) {
     protected Component createComponent() {
         AttributeSet attr = getElement().getAttributes();
         HTML.Tag t = (HTML.Tag)
-            attr.getAttribute(StyleConstants.NameAttribute);
```

Comment:
This PR has LOTS of completely unrelated changing to formatting.
Please revert ALL of it, so we can see what is related to the change and not be distracted.